django/core/mail/backends/smtp.py

```diff
@@ -57,9 +57,11 @@ def connection_class(self):
 
     @cached_property
     def ssl_context(self):
-        ssl_context = ssl.SSLContext(protocol=ssl.PROTOCOL_TLS_CLIENT)
         if self.ssl_certfile or self.ssl_keyfile:
+            ssl_context = ssl.SSLContext(protocol=ssl.PROTOCOL_TLS_CLIENT)
             ssl_context.load_cert_chain(self.ssl_certfile, self.ssl_keyfile)
+        else:
+            ssl_context = ssl.create_default_context()
```

Comment:
I don't think there is much we can do to test it :disappointed:, `ssl.create_default_context()` calls `set_default_verify_paths()` and its behavior is [not even tested](https://github.com/python/cpython/blob/6716254e71eeb4666fd6d1a13857832caad7b19f/Lib/test/test_ssl.py#L1326-L1330) in CPython.